tests/unit/App.spec.ts

```diff
@@ -15,4 +15,9 @@ describe('App.vue', () => {
     expect(app.contains(MenuRight)).toBeTruthy();
     expect(app.contains(MenuBottom)).toBeTruthy();
   });
+
+  it('window.app does not exist if not an e2e test', () => {
```

Comment:
even if window.app should exist for cypress, unit tests shouldnt care about this detail